{
  "id": "20250412234507",
  "execution_time": "2025-04-12 23:45:07",
  "env": "개발",
  "results": [
    {
      "id": "test_case_20250411_020522",
      "desc": "유효한 자격 증명으로 로그인에 성공하는지 확인합니다.",
      "result": "ERROR",
      "reason": "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000227144E8710>: Failed to establish a new connection: [WinError 10061] 대상 컴퓨터에서 연결을 거부했으므로 연결하지 못했습니다'))",
      "execution_time": "2025-04-12 23:45:07",
      "env": "개발"
    }
  ],
  "type": "ai_collection",
  "scheduled": false
}